spec/glueby/contract/timestamp_spec.rb

```diff
@@ -13,11 +13,11 @@
       [{
         'txid' => '0555c5af698db73ed6378d2ed3c71e45fc6a1dbbcf931a248a7c9221f1d3220c',
         'vout' => 0,
-        'amount' => 1.00000000
+        'amount' => 100000000
```

Comment:
This data should be a string as it comes from the Bitcoin Core RPC `listunspent`.

Reply:
I got it.
I will fix soon.